eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -197,16 +201,20 @@ extends:
                 $optional_params || true # return true always to suppress exit error logging
             displayName: 'Generate SDK'
 
-          - task: Powershell@2
-            inputs:
-              filePath: $(SdkRepoDirectory)/eng/common/scripts/Save-Package-Properties.ps1
-              arguments: >
-                -ServiceDirectory $(GeneratedSDK.ServiceName)
-                -OutDirectory $(GeneratedSDK.StagedArtifactsFolder)/PackageInfo
-              pwsh: true
-              workingDirectory: $(SdkRepoDirectory)
+          - pwsh: |
```

Comment:
Is this only for debug purpose? Can you remove it when merge this PR? Or move to the next step (ApiView-Helpers.ps1) if you want to output them?